evalml/tests/objective_tests/test_lead_scoring.py

```diff
@@ -6,7 +6,9 @@
 from evalml.objectives import LeadScoring
 
 
-def test_lead_scoring_objective(X_y_binary):
```

Comment:
This test does two things: run automl with lead scoring and test it on small data not related to automl so I'm splitting it into two

Reply:
Thanks for the explanation!